setup.py

```diff
@@ -51,7 +51,7 @@
         'pytest-cov',
         'pytest-env',
         'pytest-flake8',
-        'pytest-html',
+        'pytest-html!=2.1.0',
```

Comment:
```suggestion
        'pytest-html<2.1.0',
```

Reply:
a bit more flexible for the environment

Reply:
I would prefer specifically avoiding a single version, from looking at the discussion in the issue I suspect the issue will be solved with the next release.

Reply:
yes, but conda does prefer `<` to `!=`, anyways this is pypi so should be cool